style.css

```diff
@@ -515,3 +519,326 @@ footer {
+    width: 65%;
+    height: 756px;
+    margin-top: 120px;
+  }
+
+  .intro-heading {
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
+  }
+
+  .intro-description p {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    padding: 0 50px;
+  }
+
+  .social {
+    width: 200px;
+    height: 40px;
+  }
+
+  .social-links {
+    margin-bottom: 0;
+  }
+
+  .social-links :nth-child(1) {
+    order: 4;
+  }
+
+  .social-links :nth-child(2) {
+    order: 2;
+  }
+
+  .social-links :nth-child(3) {
+    order: 5;
+  }
+
+  .social-links :nth-child(4) {
+    order: 1;
+  }
+
+  .social-links :nth-child(5) {
+    order: 3;
+  }
+
+  .social-links a img {
+    width: 25px;
+    height: 25px;
+  }
+
+  a.action-button {
+    border: 1px solid #dfe1e6;
+    border-radius: 32px;
+    width: 40px;
+    height: 40px;
+  }
+
+  .action {
+    margin-top: 70px;
+  }
+
+  .main {
+    margin-top: 100px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 30px;
+    width: 100%;
+  }
+
+  .header-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 50%;
+  }
+
+  .header-container .indicator-container img {
+    display: none;
+  }
+
+  .works {
+    height: 100%;
+  }
+
+  .works-cards {
+    grid-template-columns: auto auto auto;
+  }
+
+  .card,
+  .text-container {
+    background-size: contain;
+    width: 369px;
+  }
+
+  .side-image {
+    display: none;
+  }
+
+  .about-me {
+    width: 100%;
+    height: 739px;
+    background-image: url('./images/illustration-about-me-desktop.png');
+    background-repeat: no-repeat;
+    justify-content: end;
+    background-position-x: right;
+    margin-top: 0;
+  }
+
+  .about-me__text {
+    width: 70%;
+  }
+
+  .form-img-center {
+    position: absolute;
+    width: 130px;
+    height: 105px;
+    left: 274px;
+    bottom: -150px;
+  }
+
+  .about-me__cards {
+    display: grid;
+    grid-template-columns: repeat(3, auto);
+    width: 100%;
+    height: 316px;
+    justify-content: center;
+  }
+
+  .about-me__card {
+    width: 369px;
+    height: 316px;
+    background-size: cover;
+  }
+
+  .about-me__header {
+    width: 50%;
+    height: 48px;
+  }
+
+  .about-me__paragraph {
+    width: 80%;
+    height: 96px;
+  }
+
+  .contact-me {
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    width: 90%;
+    height: 490px;
+    margin-bottom: 158px;
+  }
+
+  .contact-heading {
+    width: 463px;
+    height: 256px;
+    font-size: 30px;
+  }
+
+  .form-container {
+    width: 684px;
+    height: 330px;
+  }
+
+  .form-desktop-image {
+    position: absolute;
+    bottom: -170px;
+    right: -80px;
+  }
+
+  form {
+    align-items: start;
+  }
+
+  .textarea {
+    width: 100%;
+  }
+
+  .contact-details {
+    display: none;
+  }
+
+  .footer-img {
+    display: none;
+  }
+
+  footer {
+    width: 100%;
+    height: 72px;
+    justify-content: center;
+  }
+
```

Comment:
Your design looks great. Kindly include the little horizontal rule above the social icons in the footer. The background image 

Your design:
![image](https://user-images.githubusercontent.com/20554583/228991958-1925c628-d859-4391-8769-57a502690e84.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/228992035-7f1c623e-87af-487a-aee7-570da65f94e4.png)